src/__screenshot_tests__/responsive-layout-screenshot-test.tsx

```diff
@@ -0,0 +1,23 @@
+import {openStoryPage} from '../test-utils';
+
+const TEST_CASES = [
+    {id: 'layout-responsive-layout--default', device: 'MOBILE_IOS'},
+    {id: 'layout-responsive-layout--default', device: 'MOBILE_ANDROID'},
```

Comment:
I'd remove one of these. Testing iOS/Android doesn't give any value in this component, the relevant thing here is the screen size. Having different tests for Android/iOS makes sense in components that change betwen platforms, like the Switch